db/schema.rb

```diff
@@ -88,6 +88,19 @@
   add_index "bundle_products", ["bundle_product_id"], name: "fk_bundle_prod_prod", using: :btree
   add_index "bundle_products", ["product_id"], name: "fk_bundle_prod_bundle", using: :btree
 
+  create_table "card_readers", force: :cascade do |t|
```

Comment:
oh wow yeah I hadn't fully thought about what the id's actually are--do we have examples?